stable/pomerium/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- desimone
+- victornoel
```

Comment:
@victornoel Thank you for the help reviewing this (and other) PRs/ issues. You have a much better handle on helm than I do. Let me know if adding you here is okay.

Reply:
@desimone here, that's nice :) For now I'm ok being reviewer but not approver, I won't have the time to dedicate to it